wiki/markdown/usage/cli.md

```diff
@@ -10,48 +10,50 @@ $ java -jar publish-cli.jar -h
 You passed help flag.
 usage: java -jar
 
--d,--debug                        Enable debug traces.
+-d,--debug                                    Enable debug traces.
 
--domain,--domainId <arg>          Identifies the domain that produces the event.
+-domain,--domainId <arg>                      Identifies the domain that produces the event.
 
- -cc,--channelsCount <arg>        Number of channels connected to message
-                                  bus, default is 1
+ -cc,--channelsCount <arg>                    Number of channels connected to message
+                                              bus, default is 1
 
--en,--exchange_name <arg>         Exchange name.
+ -wcto,--wait_for_confirms_timeOut <arg>      Time out for wait for confirms ,default is 5sec
```

Comment:
Isn't the unit for this option 'wcto' to be entered in milliseconds? Not seconds?
It should be made clear in the usage so no confusion exists, ie example should be 5000 ms.

Reply:
yes i  will change it to  5000 ms